Sources/VaporToolbox/Run.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 // Generates an Xcode project
 struct Run: AnyCommand {
-    let help = "Runs an app from the console."
+    let help = "Runs an app from the console.\nEquivalent to `swift run --enable-test-discovery Run`."
```

Comment:
I don't think this should be wrapped in a markdown code block as Terminal won't render it like that

Reply:
I used backticks, as they signify commands that are executed in shells (which predates Markdown by decades). No Markdown here. 🤣

Do you want me to switch it for single quotes?

Reply:
Nah I'm happy with it